lib/parallel_tests/cli.rb

```diff
@@ -92,8 +93,9 @@ def report_results(test_results)
     def report_number_of_tests(groups)
       name = @runner.test_file_name
       num_processes = groups.size
-      num_tests = groups.map(&:size).inject(:+)
-      puts "#{num_processes} processes for #{num_tests} #{name}s, ~ #{num_tests / groups.size} #{name}s per process"
+      num_tests = groups.empty? ? 0 : groups.map(&:size).inject(:+)
+      tests_per_process = num_processes <= 0 ? 0 : num_tests / num_processes
```

Comment:
-1 processes seems a bit scary / I'd want that to show :D 
how about:
`tests_per_process = (num_processes == 0 ? 0 : num_tests / num_processes)`